packages/inferno-compat/src/index.ts

```diff
@@ -146,6 +146,14 @@ const injectStringRefs = function (originalFunction) {
 		if (typeof name === 'string') {
 			normalizeProps(name, props);
 		}
+
+		// React supports iterable children, in addition to Array-like
+		for (let i = 0; i < children.length; i++) {
```

Comment:
Thanks — and I appreciate knowing it is for IE. I mirrored an example elsewhere in the codebase and cached `len = children.length`.